src/metpy/calc/thermo.py

```diff
@@ -2574,3 +2574,39 @@ def specific_humidity_from_dewpoint(pressure, dewpoint):
     """
     mixing_ratio = saturation_mixing_ratio(pressure, dewpoint)
     return specific_humidity_from_mixing_ratio(mixing_ratio)
+
+
+@exporter.export
+@preprocess_xarray
+@check_units('[length]', '[temperature]', '[speed]', '[speed]')
+def gradient_richardson_number(height, potential_temperature, u, v, axis=0):
+    r"""Calculate the gradient (or flux) Richardson number.
+
+    .. math::   Ri = (g/\theta) * \frac{\left(\partial \theta/\partial z\)}
+             {[\left(\partial u / \partial z\right)^2 + \left(\partial v / \partial z\right)^2}
+
+    See [Holton2004]_ pg. 121-122. As noted by [Holton2004]_, flux Richardson number values below 0.25 indicate turbulence. 
```

Comment:
Flake8 is complaining about the line being too long and having some trailing whitespace.

Reply:
Sorry for not seeing that earlier. Fixed.